README.md

```diff
@@ -10,30 +10,41 @@ The 2nd simplest way to try is to clone [reprocessing-example](https://github.co
 
 See [below](#projects-using-reprocessing) for projects using Reprocessing!
 
+```reason;shared(sandbox)
+[@bs.val] external sandboxCanvasId: string = "";
+[@bs.val] external sandboxCanvas: 'canvas = "";
+[@bs.val] external containerDiv: 'node = "";
+[@bs.send] external addEventListener: ('node, string, 'eventT => unit) => unit = "addEventListener";
+let id = sandboxCanvasId;
+addEventListener(containerDiv, "mouseleave", (_) => Reprocessing.playPause(id, false) |> ignore);
+addEventListener(containerDiv, "mouseenter", (_) => Reprocessing.playPause(id, true) |> ignore);
+Reprocessing.setScreenId(sandboxCanvasId);
+```
```

Comment:
This is kinda ugly at the top of the README for GitHub and all.
Could you make it so that you can parse commented out chunks? So we could surround this with `<!--- stuff --->` and it wouldn't show up in other places that render this readme.

Reply:
haha oh right good call

Reply:
done!